### PR TITLE
Issue 544: Case Histories - SEG

### PR DESCRIPTION
**Pull request by dccowan**
This PR addresses issue #544 . Citations for SEG materials have been added to the case history pages. The bibtex file containing citations has also been updated.

### CODE REVIEW

**Comment by dccowan** [comment]
@lheagy I have tried to find where this is failing by locally running the python tests in the /tests directory, however they all pass. Unfortunately Travis fails. I am not entirely sure what the differences are between the tests I run locally and the tests being run on Travis. From what I understand, we specify the 'test_dir' in the travis.yml file which means the same tests are being run locally and on Tavis, correct?.

**Comment by lheagy** [comment]
okay, I will look in to it

File: content/case_histories/albany/index.rst [hunk 1]
```diff
@@ -15,7 +15,13 @@ Albany
...
+
+The material presented here has been modified from the SEG-published version and is subject to the `SEG terms of use and conditions <https://seg.org/Publications/Policies-and-Permissions/Permissions>`__. This case history was presented as a part of the `SEG 2017 Distinguished Instructor Short Course <https://seg.org/Education/Courses/DISC/2017-DISC-Doug-Oldenburg>`__.
+
+.. Thanks to the `Society of Exploration Geophysicists <http://seg.org>`__ and `Geotech Ltd. <http://www.geotech.com>`__ for permission to reproduce figures and adapt text from the source material. This case history was adapted from the expanded abstract `"Airborne and Ground Time-Domain EM results from the Albany Graphite Discovery" <http://www.cronegeophysics.com/wp/wp-content/uploads/2015/04/Albany-Graphite-Air-Ground-TEM_Expanded_abstract_final-13-Feb-2015.pdf>`__.
```
Comment (lheagy) [inline content/case_histories/albany/index.rst]: would you mind clearing out the commented out text if it is no longer needed?

File: content/case_histories/aspen/index.rst [hunk 2]
```diff
@@ -3,19 +3,24 @@
...
+
+    - Sarah Devriese and Douglas Oldenburg (2016) From exploration to reclamation: Using EM methods at SAGD sites in the Athabasca oil sands. SEG Technical Program Expanded Abstracts 2016: pp. 965-970. `https://doi.org/10.1190/segam2016-13842507.1 <https://doi.org/10.1190/segam2016-13842507.1>`__ 
+
+The material presented here has been modified from the SEG-published version and is subject to the `SEG terms of use and conditions <https://seg.org/Publications/Policies-and-Permissions/Permissions>`__. This case history was presented as a part of the `SEG 2017 Distinguished Instructor Short Course <https://seg.org/Education/Courses/DISC/2017-DISC-Doug-Oldenburg>`__.
+
+.. Thanks to the `Society of Exploration Geophysicists <http://seg.org>`__ for permission to reproduce figures and adapt text from the source material. This case study is based on the extended abstract `"From exploration to reclamation: using EM methods at SAGD sites in the Athabasca oil sands" <http://library.seg.org/doi/abs/10.1190/segam2016-13842507.1>`__ :cite:`DevrieseOldenburg2016a`.
```
Comment (lheagy) [inline content/case_histories/aspen/index.rst]: same here

File: content/case_histories/aspen/index.rst [hunk 3]
```diff
@@ -48,9 +53,8 @@ planned for the future. The readers can anticipate to learn the following:
     interpretation
     synthesis
 
-.. **References**
 
-..  .. bibliography:: ../../references.bib
-..     :style: alpha
-..     :encoding: latex+latin
-..     :filter: docname in docnames
```
Comment (lheagy) [inline content/case_histories/aspen/index.rst]: Thanks!

File: content/case_histories/albany/index.rst [hunk 4]
```diff
@@ -44,4 +46,6 @@ This case study describes the airborne time-domain EM (TDEM) and magnetic geophy
     lessons
 
 
+**Citations**
```
Comment (lheagy) [inline content/case_histories/albany/index.rst]: for only a single citation, would you mind dropping the "s" here?

File: contributors.rst [hunk 5]
```diff
@@ -463,8 +463,8 @@ Sarah G. R. Devriese
 
 .. _fefferso:
 
-Flemming Effersø
-----------------
+Flemming EffersÃ¸
```
Comment (lheagy) [inline contributors.rst]: did you intend for these to change?

File: content/references.bib [hunk 6]
```diff
@@ -302,24 +303,22 @@ @article{viezzoli2009
 number = {2},
 pages = {173-183},
 year = {2009},
-doi = {10.1071/EG08027},
-URL = {
-        http://library.seg.org/doi/abs/10.1071/EG08027
-    },
-}
```
Comment (lheagy) [inline content/references.bib]: @dccowan: I believe that the failure is due to the deleted lines above (305 - 309). Without the closed bracket here, the references file can't be parsed correctly.

File: content/case_histories/bookpurnong/index.rst [hunk 7]
```diff
@@ -3,22 +3,33 @@
...
 
     This case history follows the use of the SkyTEM airborne system to characterize hydrogeological settings.
 
 **Special Thanks**
 
-Thanks to the `Society of Exploration Geophysicists <http://seg.org>`__ and SkyTEM for permission to reproduce figures and
-adapt text from the source material. This case history is based upon two articles:
+Thanks to the `Society of Exploration Geophysicists <http://seg.org>`__ for permission to reproduce figures and adapt text from:
+
+    - Andrea Viezzoli, Esben Auken, and Tim Munday (2009). ”Spatially constrained inversion for quasi 3D modelling of airborne electromagnetic data – an application for environmental assessment in the Lower Murray Region of South Australia.” Exploration Geophysics, 40(2), 173-183. `https://doi.org/10.1071/EG08027 <https://doi.org/10.1071/EG08027>`__
```
Comment (lheagy) [inline content/case_histories/bookpurnong/index.rst]: We should also add the citation for the 2010 paper here

File: content/case_histories/bookpurnong/index.rst [hunk 8]
```diff
@@ -3,22 +3,33 @@
...
 
 .. topic :: Prelude
 
     This case history follows the use of the SkyTEM airborne system to characterize hydrogeological settings.
 
 **Special Thanks**
 
-Thanks to the `Society of Exploration Geophysicists <http://seg.org>`__ and SkyTEM for permission to reproduce figures and
-adapt text from the source material. This case history is based upon two articles:
+Thanks to the `Society of Exploration Geophysicists <http://seg.org>`__ for permission to reproduce figures and adapt text from:
```
Comment (lheagy) [inline content/case_histories/bookpurnong/index.rst]: This one is not SEG, this is Exploration Geophysics which I believe is maintained by CSIRO. @dougoldenburg: do you remember if we got explicit permission from CSIRO for this?  (Based on our acknowledgement in Mt. Isa, it looks like it)

**Pull request merged**